src/GovUk.Education.ExploreEducationStatistics.Data.Services/TableBuilderService.cs

```diff
@@ -100,6 +119,30 @@ public class TableBuilderService : ITableBuilderService
                 });
         }
 
+        private async Task<int> GetMaximumTableCellCount(ObservationQueryContext queryContext)
+        {
+            var filterItemIds = queryContext.Filters;
+            var countsOfFilterItemsByFilter = filterItemIds == null
+                ? new List<int>()
+                : (await _filterItemRepository.CountFilterItemsByFilter(filterItemIds))
+                .Select(pair =>
```

Comment:
Could just go for `.Select(pair => pair.Item2)` here, but were you just hoping to be more explicit?

Reply:
Ah yep I was trying to be more explicit about 'count' rather than returning 'Item2'.

Ideally I want to deconstruct them in the parameter so  something like `.Select((_, count) => count)` but that's not possible. Note this would  actually compile but the `int` count parameter would be assigned the index of the selector! 🙂